pkg/resourcemanager/azuresql/azuresqluser/azuresqluser.go

```diff
@@ -165,8 +165,8 @@ func (s *AzureSqlUserManager) UserExists(ctx context.Context, db *sql.DB, userna
 
 // DropUser drops a user from db
 func (s *AzureSqlUserManager) DropUser(ctx context.Context, db *sql.DB, user string) error {
-	tsql := "DROP USER @user"
-	_, err := db.ExecContext(ctx, tsql, sql.Named("user", user))
+	tsql := fmt.Sprintf("DROP USER %q", user)
+	_, err := db.ExecContext(ctx, tsql)
```

Comment:
Hong - this is risky as we are removing the sql injection checks. Why are we removing this? Is this causing the user to not get deleted? If this is not working, we should open an issue to track this down and fix this

Reply:
@janani, yes, the sql db command doesnot work because of the double quotes. I opened issue #1147 to investigate the issue for mysql and postgresql and should include azuresql user as well.  I am looking into today now.